src/commands/test.rs

```diff
@@ -0,0 +1,352 @@
+use crate::commands::utils::balance::{
+    display_balance, display_l1_balance, get_erc20_balance_decimals_symbol,
+};
+use crate::commands::utils::wallet::*;
+use crate::config::ZKSyncConfig;
+use clap::Subcommand;
+use colored::*;
+use eyre::ContextCompat;
+use spinoff::{spinners, Color, Spinner};
+use std::ops::Div;
+use zksync_ethers_rs::contracts::erc20::MINT_IERC20;
+use zksync_ethers_rs::{
+    core::{k256::ecdsa::SigningKey, rand::thread_rng, utils::parse_ether},
+    providers::{Http, Middleware, Provider},
+    signers::{LocalWallet, Signer, Wallet},
+    types::{
+        transaction::eip2718::TypedTransaction, Address, Eip1559TransactionRequest, L2TxOverrides,
+        TransactionReceipt, U256,
+    },
+    wait_for_finalize_withdrawal,
+    zk_wallet::ZKWallet,
+    ZKMiddleware,
+};
+
+#[derive(Subcommand, PartialEq)]
+pub(crate) enum Command {
+    #[clap(about = "LoadTest the zkStack Chain.")]
+    LoadTest {
+        #[clap(long = "wallets", short = 'w', required = true)]
+        number_of_wallets: u16,
+        #[clap(
+            long = "amount",
+            short = 'a',
+            required = true,
+            help = "Amount of BaseToken to deposit, 20% more will be deposited.\nThat extra 20% will remain in the main wallet,\nthe rest will be redistributed to random wallets"
+        )]
+        amount: f32,
+        #[clap(
+            long = "reruns",
+            short = 'r',
+            help = "If set to 0 it will run indefinitely, If not set defaults to 1 run."
+        )]
+        reruns_wanted: Option<u8>,
+    },
+    #[clap(about = "Mint ERC20 token on L1.")]
+    Erc20L1Mint {
```

Comment:
Why is this a command?

Reply:
Because if we have a token in a local dev environment and use it as `basetoken`, we may have the need to mint some of it.

Reply:
Ok, I get why we need to mint sometimes, but I'm not sure if this should be a test subcommand, what does this command test or how does it fit in a `test` command context? It'd be probably better to have this logic inside the corresponding test command instead of being a `test` subcommand. For example, in the `load-test` handler we could ask if we have enough ERC20 balance to start loadtesting, and if not mint it.

What do you think?

Reply:
Done, now it asks if the balance is enough, if not, it will mint some tokens. Maybe we can rewrite an `erc20-l1-mint` subcommand for the `utils` cmd if we need it.